web_m2x_options/static/src/js/form.js

```diff
@@ -78,41 +118,19 @@ odoo.define('web_m2x_options.web_m2x_options', function (require) {
             this._super.apply(this, arguments);
         },
     });
-
+   
     FieldMany2One.include({
 
-        start: function () {
+        _bindAutoComplete: function () {
```

Comment:
Looks like you are overriding this obscure, private method because it is only called for widgets in edit mode, and then only once per widget. Maybe you can document this (or for whatever other reason you are changing this) in a docstring?